src/main/java/com/google/sps/taskqueue/push/BlobstoreTaskWorker.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.taskqueue.push;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/blobstoreWorker")
```

Comment:
More clear name. A "worker" could be anything.

Reply:
I use Worker because that is how they describe the servlet in https://cloud.google.com/appengine/docs/standard/java/taskqueue/push. Should I change it to like blobstoreDeleteWorker? I think that's more specific.